spring-pulsar/src/test/java/org/springframework/pulsar/observation/ObservationIntegrationTests.java

```diff
@@ -62,7 +62,7 @@
  * @author Chris Bono
  * @see SampleTestRunner
  */
-public class ObservationIntegrationTests extends SampleTestRunner {
+public class ObservationIntegrationTests extends SampleTestRunner implements PulsarTestContainerSupport {
```

Comment:
Now tell me that it wasn't cool (and logical) to migrate a Pulsar container into an interface! 😄

Reply:
Yeh this breaks the artificial "Is A" relationship. Cleans things up. Thanks for pointing it out. BTW... I think I may have done this in the past and forgot completely about it based on the [author of this one??](https://github.com/spring-projects/spring-integration/commit/26816a3eef5d7b6ca69136bb777e49a5646dcc22#diff-91b1012a2c92f870457df7e4a12d79c545a804d7eb8fc02b9aab6874278d4026R28).

Reply:
No, that was done already without you: my personal request in the respective PR: https://github.com/spring-projects/spring-integration/pull/3895#discussion_r980199193 😄

Reply:
Hehe. Thats good, because otherwise this would be a huge case of "I really do not remember doing that code".